core/state/trie_prefetcher.go

```diff
@@ -42,8 +52,14 @@ type triePrefetcher struct {
 	fetches  map[common.Hash]Trie        // Partially or fully fetcher tries
 	fetchers map[common.Hash]*subfetcher // Subfetchers for each trie
 
-	abortChan chan *subfetcher
-	closeChan chan struct{}
+	closed            int32
```

Comment:
I am confused about how `fetches` here been used, I did not see anywhere it is been assigned.

Reply:
`fetches ` is for copied triePefetcher only, for newly created triePrefetcher, `fetches` is nil and useless